### ISSUE TITLE
Readme request: Provide examples of values to use in the timeMin and timeMax fields.

### ISSUE THREAD

**Issue by YousufSSyed** [dustinksi/obsidian-sync-calendar#6]
From looking at the readme, I'm not sure what else I can enter in those fields other than specific dates, other than to leave them unspecified to instead use the default values. It does mention moment.js but I didn't find any documentation pages that I think are or would be relevant for this plugin. 

What about moment.js syntax applies to this plugin? Can all the relevant documentation page(s) for moment.js be linked in the readme? Can you provide more examples of possible `timeMin` and `timeMax` values with moment.js syntax?

**Comment by dustinksi** [comment]
Hi! I’ve updated the README. Thank you for your efforts to enhance the plugin's documentation.

**Comment by YousufSSyed** [comment]
Thanks for adding that. It looks good but I'm still wondering about relative dates for `timeMin` and `timeMax`, since the defaults for those are "One week before the current time" and "null" respectively. 
* How could I set the dates for either field to be today's date
* How can I make them x number of days, months, weeks, years, etc. past or after the current date?

**Comment by dustinksi** [comment]
You're right, if I understand correctly, you would like to use relative time settings in Markdown code blocks to filter events. Unfortunately, this feature is not currently implemented, i.e., `timeMin` and `timeMax` can only be set up as an absolute timestamp via Markdown code blocks.

However, this is a good requirement and worth implementing as soon as possible. At the moment, the functionality to grab time ranges (weeks ago) is also rather weak. 

I will try to find time before next week to implement the above features, before I submit the new features, you can try filtering events directly using absolute timestamps, like:

````
```sync-calendar
name: "{numberTodos} todos @ Apr. 21's noon"
timeMin: "2023-04-21 11:30"
timeMax: "2023-04-21 14:00"
```
````
